### PR TITLE
Remove rmw_connext_cpp references.

### PR DESCRIPTION
**Pull request by clalancette**
Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>

### CODE REVIEW

File: rclcpp/test/rclcpp/executors/test_static_executor_entities_collector.cpp [hunk 1]
```diff
@@ -267,7 +267,7 @@ TEST_F(TestStaticExecutorEntitiesCollector, add_remove_node_with_entities) {
   auto client = node->create_client<test_msgs::srv::Empty>("service");
   auto waitable = std::make_shared<TestWaitable>();
 
-  // Adding a subscription with rmw_connext_cpp adds another waitable, so we need to get the
+  // Adding a subscription with could add another waitable, so we need to get the
```
Comment (ivanpauno) [inline rclcpp/test/rclcpp/executors/test_static_executor_entities_collector.cpp:270]: there's something missing between `with` and `could`, maybe

```suggestion
  // Adding a subscription could add another waitable, so we need to get the
```

**Review by ivanpauno — APPROVED** [review]

**Review by ivanpauno — APPROVED** [review]
should be merged after https://github.com/ros2/ros2/pull/1116 as well

**Review by asorbini — APPROVED** [review]
LGTM as a follow up to #1574

**Review by wjwwood — APPROVED** [review]

**Comment by clalancette** [comment]
Green CI is at https://github.com/ros2/rmw_connextdds/issues/9#issuecomment-806840458

**Pull request merged**